### DISCUSSION TITLE
import passwords

### DISCUSSION THREAD

**Discussion by ferrvittorio** [minbrowser/min#2453] [Q&A]
Hi, is there any way to import passwords from chrome?

**Comment by Tedd-Codes** [comment]
Maybe let me look into it

**Reply by Tedd-Codes** [reply]
There currently isnt a way

**Reply by Tedd-Codes** [reply]
Sorry, there (to my knowledge) inst a way to import passwords from different browsers but if you are using the newest/current version. (Witch you probably are) or not a old version there is a way to export passwords from chrome. So i will look at the format and find a way to try to add it. if you would give me any info on the ui(User interface) you would like for the import then i could try to make it.
